src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.cinema.hall.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.cinema.hall.dto.CinemaHallResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.mapper.CinemaHallDtoMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema/halls")
+public class CinemaHallController {
+    private CinemaHallService cinemaHallService;
+    private CinemaHallDtoMapper cinemaHallDtoMapper;
+
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallDtoMapper cinemaHallDtoMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallDtoMapper = cinemaHallDtoMapper;
+    }
+
+    @PostMapping
+    public CinemaHallResponseDto add(@RequestBody CinemaHallRequestDto cinemaHallRequestDto) {
+        return cinemaHallDtoMapper
+                .toDto(cinemaHallService.add(cinemaHallDtoMapper.toModel(cinemaHallRequestDto)));
+    }
+
+    @GetMapping("/")
```

Comment:
you don't need put empty path parameters:
```suggestion
    @GetMapping
```